apps/meteor/app/livechat/server/business-hour/Helper.ts

```diff
@@ -52,8 +53,10 @@ export const openBusinessHourDefault = async (): Promise<void> => {
 		},
 	});
 	const businessHoursToOpenIds = (await filterBusinessHoursThatMustBeOpened(activeBusinessHours)).map((businessHour) => businessHour._id);
+	businessHourLogger.debug('Opening default business hours', businessHoursToOpenIds);
```

Comment:
even though the other syntax works, it is a hack since we have legacy code still using it..

we must avoid using multiple params for logs as it doesn't work exactly as `console.log` https://github.com/pinojs/pino/blob/master/docs/api.md#interpolationvalues-any
```suggestion
	businessHourLogger.debug({ msg: 'Opening default business hours', businessHoursToOpenIds });
```